bin/ch/WScriptJsrt.cpp

```diff
@@ -600,30 +600,6 @@ JsValueRef WScriptJsrt::LoadScriptHelper(JsValueRef callee, bool isConstructCall
     return returnValue;
 }
 
-JsErrorCode WScriptJsrt::InitializeModuleInfo(JsModuleRecord moduleRecord)
```

Comment:
Drive by clean up in the host with past changes to module logic there was no reason for this to be a separate function any more - it's a one off initialisation step.